service/controller/aws/v7patch2/resource/chartconfig/create.go

```diff
@@ -7,9 +7,9 @@ import (
 	"github.com/giantswarm/microerror"
 	"github.com/giantswarm/operatorkit/controller/context/reconciliationcanceledcontext"
 
-	"github.com/giantswarm/cluster-operator/pkg/v7patch1/chartconfig"
-	"github.com/giantswarm/cluster-operator/pkg/v7patch1/key"
```

Comment:
Here it was `v7patch1`. Above are lots of `v7`. Why is that?

Reply:
IMO the previous patch (`v7patch1`) did not replace imports correctly and it was using `v7` instead